vocabularies/Org.OData.Capabilities.V1.sample.xml

```diff
@@ -0,0 +1,513 @@
+<root>
```

Comment:
Done! 
I also scoped the sample down to permissions since that's what the PR is addressing. 
For 'CustomerQueryOptions' and 'CustomHeaders', I will create a separate example to demonstrate usage.